Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -9495,7 +9495,22 @@ public bool ClickAndValidteHandler(ActUIElement act)
 
             if (elmToValidate != null)
             {
-                return true;
+                bool validationResult = false;
```

Comment:
Create local function add call it from both the places

Reply:
done